src/accountant/core.cljs

```diff
@@ -83,7 +83,7 @@
            host (.getDomain uri)
            current-host js/window.location.hostname]
        (when (and (not any-key) (= button 0) (path-exists? path) (= host current-host))
-         (set-token! history relative-href title)
+         (. history (setToken relative-href title))
```

Comment:
It's javascript interop, sits on the `history` object. I would have preferred
`(.setToken history relative-href title)`
but tried to follow the style used in the `navigate!`function.